frontend/src/routes/(app)/(home)/[user]/projects/+page.svelte

```diff
@@ -5,9 +5,13 @@
 	export let data;
 
 	$: ownProjects =
-		data.user === null ? [] : data.projects.filter((proj) => proj.ownerName === data.user?.name);
+		data.user === null
+			? []
+			: data.projects.filter((proj) => proj.project.ownerName === data.user?.name);
```

Comment:
Would probably rename `projects` to projectDetails` to make it more clear. Bit weird to have `project` inside `projects`.